Assets/Scripts/UI/DialogBox/DialogBoxNewGame.cs

```diff
@@ -29,6 +30,9 @@ public override void ShowDialog()
         Width.onEndEdit.AddListener(delegate { VerifyNumericInput(Width); });
         Depth.onEndEdit.AddListener(delegate { VerifyNumericInput(Depth); });
 
+        // Generate Random Seed TODO: make it alphanumerical
+        Seed.text = UnityEngine.Random.Range(0, int.MaxValue).ToString();
```

Comment:
This can be done by hardcoding in every alphanumeric character and using rng to select them or by using 
`Path.GetRandomFileName().Replace(".", "")`

of course keeping it as a integer is fine for now.